dev/io.openliberty.xmlBinding.3.0.internal.tools/jaxb_schemagen.bnd

```diff
@@ -16,8 +16,8 @@ Bundle-Name: IBM JAXB SchemaGen Tool
 
 Require-Compiler: true
 
-Require-Bundle: io.openliberty.jaxb.3.0.internal.tools; version="[1,1.0.100)", \
- io.openliberty.jakarta.jaxb.3.0; version="[1,1.0.100)"; location="dev/api/spec/", \
+Require-Bundle: io.openliberty.xmlBinding.3.0.internal.tools; version="[1,1.0.100)", \
+ io.openliberty.jakarta.xmlBinding.3.0; version="[1,1.0.100)"; location="dev/api/spec/", \
  com.ibm.ws.logging; version="[1,1.0.100)"
  
 Command-Class: io.openliberty.jaxb.tools.SchemaGen
```

Comment:
Should the package name change to be xmlbinding instead of jaxb?

Reply:
I can give it try